README.md

```diff
@@ -25,7 +25,7 @@
 
 > # Bookstore CMS
 
-| Desktop Veiw Representation|
+| Desktop View Representation|
 |---------------------------------------|
 |<div align="center"><img src="./project-image.png" alt="screenshot" width="auto" height="auto"/></div>|
 |<div align="center"><img src="./project-image.png" alt="screenshot" width="auto" height="auto"/></div>|
```

Comment:
- [ ] The links to your screenshot images on your readme file are broken. This is probably because the images are not in the repository hence the pathname causing some issues. Kindly add the images or remove the links to keep your repo clean and professional